Control/DeepSeq.hs

```diff
@@ -327,12 +373,16 @@ instance NFData Word64   where rnf = rwhnf
 #if MIN_VERSION_base(4,7,0)
 -- |@since 1.4.0.0
 instance NFData (Proxy a) where rnf Proxy = ()
+instance NFData1 Proxy    where liftRnf _ Proxy = ()
```

Comment:
I'm not sure if it's necessary, as that's implied by the fact `NFData1` didn't exist before `1.4.3.0` anyway. But I'll add them anyway.